RNWCPP/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -225,6 +247,22 @@ XamlView ViewViewManager::CreateViewControl(int64_t tag)
     }
   });
 
+  contentControlPtr->AccessibilityInvoke([=]()
+  {
+    auto instance = m_wkReactInstance.lock();
+    if (instance != nullptr)
+    {
+      auto pNativeUiManager = static_cast<NativeUIManager*>(instance->NativeUIManager());
+      facebook::react::INativeUIManagerHost* pUIManagerHost = pNativeUiManager->getHost();
+
+      auto pViewShadowNode = static_cast<ViewShadowNode*>(pUIManagerHost->FindShadowNodeForTag(tag));
+      if (pViewShadowNode != nullptr && pViewShadowNode->OnClick())
```

Comment:
I know the ShadowNode should be there, but do we really want to hit the else statement if it's not?

Reply:
for the other events we don't check that, all we need is a tag